src/plugins/variables-plugin.tsx

```diff
@@ -147,12 +154,17 @@ export function VariablesPlugin(variables: Record<string, number>): HtmlSerializ
         }
         if (reference) {
           name = reference;
-          value = formatVariable(variables[name]);
+          value = formatVariable(presetVariables[name]);
+        }
+        if (editor.value.selection) {
+          editor.moveToEnd()
+                .insertText(" ");
```

Comment:
While playing with this more I found that if I double click on a chip and then hit OK, a space gets added before the chip. I'm not really sure this is worth fixing though.